praw/models/reddit/mixins/messageable.py

```diff
@@ -1,11 +1,19 @@
 """Provide the MessageableMixin class."""
+from typing import Optional, TypeVar
 from ....const import API_PATH
 
+Subreddit = TypeVar("Subreddit")
+
 
 class MessageableMixin:
     """Interface for classes that can be messaged."""
 
-    def message(self, subject, message, from_subreddit=None):
+    def message(
+        self,
+        subject: str,
+        message: str,
+        from_subreddit: Optional[Subreddit] = None,
```

Comment:
This can be a `Subreddit` object but it could also be a string. (see docstring below).

Reply:
That would be correct.